x-pack/plugins/session_view/public/components/detail_panel_alert_actions/index.tsx

```diff
@@ -58,7 +61,7 @@ export const DetailPanelAlertActions = ({
     return null;
   }
 
-  const { uuid } = event.kibana.alert;
+  const { uuid } = event.kibana?.alert ?? {};
```

Comment:
why fall backing to an empty object? shouldn't be an empty string instead?

Reply:
After discussing, uuid could still be undefined if the object is `{ event: { kibana: { alert: { uuid: undefined } } } }` i will add `uuid ?? ''` below when it's being used